config/docker/notify.sh

```diff
@@ -3,16 +3,35 @@
 ## $template contains the template to execute
 ## $verb
 
+function set_up_configs {
+  cp $file $dst_file
+  if [ -n "$IFACE" ]; then
+    sed -i "s/interface.*$/interface $IFACE/g" $dst_file
+    echo "autodicovered local interface that can reach $reachip to be $IFACE"
+  fi
+}
+
 set -o nounset
 set -o errexit
 
 HASH=$(md5sum $(readlink -f $file))
 
+IFACE=""
+if [ -n "$reachip" ]; then
+  IFACE=$(ip route get $reachip | awk "/$reachip/{ print \$3 }")
+fi
+
+if [ "$setup" = "true" ]; then
```

Comment:
can we call this `create_config_only`?

Reply:
> 
> 
> @raffaelespazzoli I should have included all the requested changes. Let me know if it looks good now
> Regarding the `set -o nounset` question, I don't believe it should interfere, since the function is only called after the `set` directive, it should still benefit from checking for undefined variables (and my tests seemed to confirm this). However, I am also a bit rusty on bash so I'm not 100% sure.

I receive
```
/usr/local/bin/notify.sh: line 20: create_config_only: unbound variable
```
found on openshift in logs of container `config-reloader`